ArduPlane/tiltrotor.cpp

```diff
@@ -572,13 +581,28 @@ void Tiltrotor::vectoring(void)
         // we need to use the same factor here to keep the same roll
         // gains when tilted as we have when not tilted
         const float avg_roll_factor = 0.5;
-        const float tilt_offset = constrain_float(yaw_out * cos_tilt + avg_roll_factor * roll_out * sin_tilt, -1, 1);
+        const float tilt_offset = (throttle_scaler * yaw_out * cos_tilt + avg_roll_factor * roll_out * sin_tilt) * yaw_range;
+
+        float left_tilt = base_output + tilt_offset;
+        float right_tilt = base_output - tilt_offset;
+
+        // if tilt offset results in output saturation set yaw limit flag
+        // Set flag on both forward and backward tilt limit
+        // this could result in less yaw output in hover as rear limit is hit first
+        // however in that case more yaw effort results in a net forward thrust
+        if ((left_tilt > 1.0) || (left_tilt < 0.0) ||
+            (right_tilt > 1.0) || (right_tilt < 0.0)) {
```

Comment:
Could change this to only set limit flag if both tilts are saturated, that would be a smaller change in behaviour.

Reply:
probably do need to make it both saturated, as some vehicles have very little (or no?) back tilt and rely on pitch

Reply:
done